src/components/ErrorShare.jsx

```diff
@@ -0,0 +1,22 @@
+import styles from '../styles/emptyAndError'
+
+import React from 'react'
+import { translate } from 'cozy-ui/react/I18n'
+
+export const ErrorShare = ({ t, errorType }) => {
+  return (
+    <div className={styles['pho-errorShare']}>
+      <h2>{ t(`Error.${errorType}_title`) }</h2>
+      <p>{ t(`Error.${errorType}_text`)}</p>
+    </div>
+  )
+}
+
+const TranslatedError = translate()(ErrorShare)
+
+export const withError = (onError, type, BaseComponent) => props =>
```

Comment:
I was actually thinking that there's some refactoring to do, there could be only one component to deal with Empty, Error, Unshared states but i'm not feeling like doing it as i'm not HOC friendly :p